bears/php/PHPMessDetectorBear.py

```diff
@@ -0,0 +1,39 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='phpmd',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+)\s*(?P<message>.*)')
```

Comment:
do not capture filename, as it is useless information that the bear already has.  (please check our docs dont recommend capturing filename)

and don't use `.*` unless there is no other option.